lib/github_changelog_generator/parser_file.rb

```diff
@@ -0,0 +1,36 @@
+module GitHubChangelogGenerator
+  class ParserFile
+    def initialize(options)
+      @options = options
+    end
+
+    def file
+      File.expand_path(@options[:params_file] || ".github_changelog_generator")
+    end
+
+    def has_file?
+      File.exist?(file)
+    end
+
+    def file_open
+      File.open(file)
+    end
+
+    def parse!
+      return false unless has_file?
```

Comment:
Why `false`, not  just `nil`?
I believe it's better to return only one type (`Hash` in this case) from method or `nil`. It's MHO. I didn't find any guidelines about that.  Thoughts?